Realm.xcodeproj/project.pbxproj

```diff
@@ -85,6 +85,10 @@
 		3F512DAE19F9A1F300746016 /* SwiftUnicodeTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = E891759A197A1B600068ACC6 /* SwiftUnicodeTests.swift */; };
 		3F512DAF19F9A1F300746016 /* TransactionTests.m in Sources */ = {isa = PBXBuildFile; fileRef = E81A1FD11955FE0100FDED82 /* TransactionTests.m */; settings = {COMPILER_FLAGS = "-fobjc-arc-exceptions"; }; };
 		3F512DB019F9A1F300746016 /* UnicodeTests.m in Sources */ = {isa = PBXBuildFile; fileRef = E8917597197A1B350068ACC6 /* UnicodeTests.m */; settings = {COMPILER_FLAGS = "-fobjc-arc-exceptions"; }; };
+		3F68999B1A9FB04800DFDE6A /* EncryptionTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 3F68999A1A9FB04800DFDE6A /* EncryptionTests.m */; };
```

Comment:
Looks like only the `EncryptionTests.m` changes are relevant in here. It'd be nice to clean this up a bit.